packages/api/src/apps/api/app.ts

```diff
@@ -140,17 +140,19 @@ export default async (env: ProcessEnv) => {
     globalStats: Services.stats.Global({ debug }, appState),
   };
 
+  const initBlock = await provider.getBlock("latest");
+
   // warm caches
-  await services.registry();
+  await services.registry(appState.lastBlockUpdate, initBlock.number);
```

Comment:
these calls take in a start block and end block. just explicitly setting the range on all these calls to be sure all events are captured